<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OverviewLayoutWidget</class>
 <widget class="QWidget" name="OverviewLayoutWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="layoutButtonWidget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="grRadioButton">
        <property name="toolTip">
         <string>Plot longest GR log. Disabled if no GR log exists</string>
        </property>
        <property name="text">
         <string>Longest gamma ray (GR) log</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="longestLogRadioButton">
        <property name="toolTip">
         <string>Plots longest log of any type</string>
        </property>
        <property name="text">
         <string>Longest log</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="specifyDataRadioButton">
        <property name="toolTip">
         <string>Select specific data to display</string>
        </property>
        <property name="text">
         <string>Select data to display</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditLongestLog">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditLongestGRLog">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="selectDataWidget" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="1">
       <widget class="QComboBox" name="typeComboBox">
        <property name="toolTip">
         <string>Data type (eg GR log type)</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="dataComboBox">
        <property name="toolTip">
         <string>Data (eg specific GR log)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="DataTypeLabel">
        <property name="text">
         <string>Data type</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="dataClassComboBox">
        <property name="toolTip">
         <string>Data class (eg Well log)</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="dataLabel">
        <property name="text">
         <string>Data</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="dataClassLabel">
        <property name="text">
         <string>Data class</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>17</width>
       <height>114</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
